utils/driver_options/driveroptions.cpp

```diff
@@ -6,20 +6,110 @@
 #include <string>
 #include <iostream>
 #include <sstream>
+#include <vector>
+#include <map>
 #include "utils/options/options.hpp"
 #include "utils/driver_options/driveroptions.hpp"
 
-
 /**
  * @brief      Extracts the relevant driver options from the config into class variables.
  */
 DriverOptions::DriverOptions() {
     Options::parse_config_file();
 
-    devices_ = config_pt.get<std::string>("devices");
+    devices_ = config_pt.get<std::string>("device_options");
+    
+    auto n200_list = config_pt.get_child("n200s");
+    // These maps are sorted by their keys (device number / int antenna number)
+    std::map<uint32_t, std::string> devices_map;    // Maps device number to IP address
+    std::map<uint32_t, bool> rx_map;                // Maps device number to rx flag
+    std::map<uint32_t, bool> tx_map;                // Maps device number to tx flag
+    std::map<uint32_t, uint32_t> int_antenna_map;   // Maps interferometer antenna number to device number
+
+    // Count up antennas while looping through N200 array
+    main_antenna_count_ = 0;
+    interferometer_antenna_count_ = 0;
+
+    // Iterate through all N200s in the json array
+    for (auto n200 = n200_list.begin(); n200 != n200_list.end(); n200++)
+    {
+        // Start iterator on first item (addr)
+        auto iter = n200->second.begin();
+
+        // Get n200 IP address
+        auto addr = iter->second.data();
+
+        // Get rx, tx, and rx_int flags
+        iter++;
```

Comment:
This block of code tells me that the order of the fields within an n200 field in the config.ini matters. Is this clarified in the documentation somewhere? It might be nice to have it NOT matter, so that the fields can be in whichever order and this code can parse them correctly regardless.